qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
Maintenance mode disable login to allow us to change things on teh site, so why would we want people changing passwords and doing other user db changes when the site is in maintenance mode?